### DISCUSSION TITLE
How does power_on_beahvior work?

### DISCUSSION THREAD

**Discussion by Awebian** [Koenkk/zigbee2mqtt#13934] [Devices]
I am trying to find out how this is supposed to work to see if I have an issue or not.  

I set this to "previous" on two of my Ikea TRÅDFRI lights.  Then I went to the lights that were both off from a zigbee state, not physically powered off.  I unplugged the light to simulate a power outage and then plugged it back in and it turned ON?  Seems it did not remember its previous state of being OFF.  

I thought that this was a feature of the bulb and settings were saved in the bulb so it remembers its previous state?

Also, I thought Phillips Hue bulbs had this feature and the setting is missing for all of my Phillips Hue lights?

**Comment by sjorge** [comment]
That is how it should work yes, are your tradfri lights up to date?
My GU10 and FLOAT panels at least follow the power_on_behavior just fine.
